docs/DeveloperGuide.md

```diff
@@ -1479,10 +1491,112 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+## Clearing CLI-nic
+
+1. Clear command format: `clear`
+
+   1. Test case: Clear command with no additional arguments e.g. `clear`<br>
+      Expected: CLI-nic clears all suppliers and warehouses data in CLI-nic.
+   1. Test case: Clear command with additional arguments e.g. `clear test` or `clear i/1`<br>
+      Expected: Similar to previous.
+
+### Editing a Supplier
+
+1. Edit command format: `edit ct/s i/INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/REMARK]`
+
+   1. Prerequisites: Suppliers in CLI-nic does not have a supplier named Alice Ptd Ltd (with the exception of test case to test for duplicated supplier).
+
+   1. Test case: Minimal information e.g. `edit ct/s i/1 n/Alice Pte Ltd`<br>
+      Expected: Edits a supplier in index 1 on supplier list to have a name "Alice Pte Ltd".
+   1. Test case: With all fields supplied e.g. `edit ct/s i/1 n/Alice Pte Ltd p/90345623 e/alice@gmail.com r/First Supplier`<br>
+      Expected: Edits a supplier in index 1 on supplier list with all the fields applied.
+   1. Test case: Invalid Prefix or missing compulsory Prefixes e.g. `edit ct/s i/1 n/Alice Pte Ltd p/90345623 e/alice@gmail.com z/large `
+      or `edit ct/s i/1`<br>
+      Expected: No supplier is added. Error details shown in the response message. A help message for edit command will also be displayed
```

Comment:
I think it should be "no supplier is edited"!

Reply:
Oh yes, you are right! Thanks for spotting it :)